package.json

```diff
@@ -4861,6 +4861,30 @@
 				"category": "GitLens",
 				"icon": "$(globe)"
 			},
+			{
+				"command": "gitlens.copyDeepLinkToBranch",
+				"title": "Copy Deep Link to Branch on Remote",
```

Comment:
Let's nuke the "Deep" and "on Remote" parts of these.
So this one would be `Copy Link to Branch`

I'm a little tempted to call it `Copy GitLens Link to Branch` because we will soon need to add a `Copy GitKraken Link to Branch`. Though it might be better to keep it as 1 or maybe two actions: `Copy Link to Branch` and `Copy Link to Branch [in|via|something]...` where the `...` version will prompt you for the target -- `GitLens`, `GitKraken Client`, `GitKraken.dev`, etc. And the non-`...` version would default to `GitLens`, but could be changed with a setting